cfpb/retirement
# Replace retirement tool hero with intro text and image

Update style of retirement tool intro based on its new place in the page hierarchy after addition of retirement portal page


## Additions

- New Spanish and English circular images
- Breadcrumbs & associated less file

## Changes

- Replace hero with intro text and circular image
- Update less for intro

## Testing

- Check out branch
- Run `gulp styles`
- Navigate to English and Spanish versions of retirement tool and verify that intro matches screenshots


## Screenshots

![screencapture-localhost-8000-retirement-before-you-claim-1505235907238](https://user-images.githubusercontent.com/778171/30339207-1eab1412-97a3-11e7-94f0-9219fea054c3.png)
![screencapture-localhost-8000-retirement-before-you-claim-1505236399454](https://user-images.githubusercontent.com/778171/30339217-265023ba-97a3-11e7-8d2e-c84c87bdb1f2.png)

![screencapture-localhost-8000-retirement-before-you-claim-es-1505235918420](https://user-images.githubusercontent.com/778171/30339213-2336c8be-97a3-11e7-9c59-71666ec7f131.png)
![screencapture-localhost-8000-retirement-before-you-claim-es-1505236382341](https://user-images.githubusercontent.com/778171/30339214-242d5fda-97a3-11e7-918f-fee395fd526a.png)


## Checklist
